source/docs/essentials/extend-the-theme.md

```diff
@@ -1,6 +1,129 @@
 ---
+id: extend-the-theme
 title: Extend the theme
 ---
 
-> Content coming soon. See
-> [https://gitlab.com/front-commerce/front-commerce/issues/13](https://gitlab.com/front-commerce/front-commerce/issues/13)
+## Understanding theme overrides
+
+Front-Commerce web application is a fully featured e-commerce universal React application
+aimed at providing a sane base for your own theme.
+Even though you could reuse pages and components individually in a totally different theme,
+most of the time you might find easier and faster to start with the base theme and
+iterate from there.
+
+**Theme override is the mechanism allowing to extend, adapt, rewrite almost everything contained in the base theme.** It is what allows you to:
+- upgrade Front-Commerce and benefit from the latest component improvements,
+- customize how an extension is displayed,
+- or create a slightly different theme for the Black Friday in confidence!
+
+People having an understanding of [layout overrides in Magento](https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/layouts/layout-override.html), [child themes in Wordpress](https://developer.wordpress.org/themes/advanced-topics/child-themes/) / [Prestashop](https://devdocs.prestashop.com/1.7/themes/reference/template-inheritance/parent-child-feature/), [templates override in Drupal](https://www.drupal.org/docs/8/theming/twig/working-with-twig-templates), [themes in CakePHP](https://book.cakephp.org/3.0/en/views/themes.html) or other similar implementations should easily understand this mechanism in Front-Commerce.
```

Comment:
I don't really know how to turn the phrase around, but I'm not a big fan of the current phrase because it tells people how they should feel.

Maybe something along those lines: "Having an understanding of ... will help you understand Front-Commerce's theme overrides because they all share the same spirit. But if you have no previous experience with these kind of features, let's see how it works!"